ucsb-cs156-f24/team02-f24-02
# Leozhu0 Menu Items Edit Page

Create Page for menu items is complete. Please merge https://github.com/ucsb-cs156-f24/team02-f24-02/pull/98 first. The link to view the storybook is as follows: https://6726c65fac869bba81d3d68d-walhwaqmyd.chromatic.com/?path=/docs/pages-ucsbdiningcommonsmenuitems-ucsbdiningcommonsmenuitemseditpage--docs

To test:
1. Run mvn spring-boot:run and npm start
2. Navigate to http://localhost:8080/ucsbdiningcommonsmenuitems/create and add a menu item
3. After adding an item, the table should populate with that item.
4. Sign in with an admin account to view Edit and Delete buttons
5. Click Edit, it should bring you to the edit page. Make any valid change and see the change reflected in the index page.

<img width="1366" alt="Screenshot 2024-11-10 at 9 56 04 PM" src="https://github.com/user-attachments/assets/1c992ba2-246c-430d-9b9d-b01e73d055da">
